.github/workflows/ci.yml

```
@@ -24,6 +24,22 @@ jobs:
       - name: Run tests
         run: pytest
 
+  docker-pytest:
+    runs-on: ubuntu-latest
+    needs: build
+    timeout-minutes: 90
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)